apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-form.tsx

```diff
@@ -14,11 +15,11 @@ export const VocabularyListForm = ({
     onFormChange,
     formState,
 }: VocabularyListFormProps): JSX.Element => (
-    <Card>
+    <VocabularyListWrapper>
         <CardHeader title="Filter the Vocabulary List" />
-        <CardContent>
+        <CardContent sx={{ textAlign: 'center' }}>
             {/* TODO [https://www.pivotaltracker.com/story/show/184066412] Use `DynamicForm` for this */}
-            <FormControl>
+            <FormControl style={{ display: 'inline-block' }}>
```

Comment:
What's the difference between sx and style?

Reply:
sx is specifically MUI, but also inline, so maybe we'd develop this purely in styled components?